pkg/infrastructure/openstack/preprovision/securitygroups.go

```diff
@@ -64,7 +64,16 @@ func SecurityGroups(ctx context.Context, installConfig *installconfig.InstallCon
 			return fmt.Errorf("failed to create the Control plane security group: %w", err)
 		}
 
-		if err := attributestags.Add(ctx, networkClient, "security-groups", masterGroup.ID, "openshiftClusterID="+infraID).ExtractErr(); err != nil {
+		// The Neutron call to add a tag
+		// (https://docs.openstack.org/api-ref/network/v2/#add-a-tag)
+		// doesn't accept all special characters. Here we use the
+		// "replace-all-tags" call instead, because it accepts a more
+		// robust JSON body.
+		//
+		// see: https://bugzilla.redhat.com/show_bug.cgi?id=2299208
```

Comment:
fyi, this is the downstream issue tracker. You should probably link to bug on Launchpad instead https://bugs.launchpad.net/neutron

Reply:
The issue comes from a middleware doing path splitting. I had a chat with the downstream Neutron team after which I wasn't sure whether the middleware was shipped with RHOSP or if it was part of OpenStack itself